bin/dynamic_challenge_score.py

```diff
@@ -335,12 +358,14 @@ def calculate_all_scores(
 
                 if score_metric == "forecast":
                     scores = forecast(truth, pred, system)
-                else:
+                elif score_metric == "reconstruction":
                     scores = (reconstruction(truth, pred),)
+                else:
+                    scores = house_zero_score(truth, pred)
 
                 for key, index in zip(score_keys, score_indices):
                     # set the score to 0 if negative
```

Comment:
Ah, I forgot to remove this comment. Can you help me remove it as it's not the case here anymore? 
Thank you.